### ISSUE TITLE
Crash under remote desktop

### ISSUE THREAD

**Issue by stuarteberg** [ilastik/ilastik#1297]
A user reported on the mailing list that using NoMachine (a remote desktop viewer) in combination with ilastik on Linux apparently causes a crash in the latest version of ilastik:

>we experienced a crash of Ilastik in the "Pixel Classification"
workflow, as soon as the "Feature Selection" tab is clicked. The error
occured on 2 independent linux systems, one with Ubuntu 15.10, the other
one with Ubuntu 14.04.
The systems were used in a remote desktop session using the software
NoMachine and virtual desktops. The previous Ilastik versions were fine
with that setup.

He attached the following console output, which shows a segfault at the end. :-(

```
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0x5000172
QGLContext::makeCurrent(): Failed.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0x5000172
QGLContext::makeCurrent(): Failed.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0x5000172
QGLContext::makeCurrent(): Failed.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0x5000172
QGLContext::makeCurrent(): Failed.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0x5000172
QGLContext::makeCurrent(): Failed.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: BadMatch (invalid parameter attributes) 8
  Extension:    128 (Uknown extension)
  Minor opcode: 11 (Unknown request)
  Resource id:  0x5000289
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0x5000172
QGLTempContext: Unable to create GL context.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0xf3
QGLContext::makeCurrent(): Failed.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 5 (Unknown request)
  Resource id:  0x50002df
QGLTempContext: Unable to create GL context.
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 112 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 115 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 114 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 117 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 1 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 139 (Unknown request)
  Resource id:  0x5000172
X Error: GLXBadContextTag 132
  Extension:    128 (Uknown extension)
  Minor opcode: 145 (Unknown request)
  Resource id:  0x5000172
[xcb] Too much data requested from _XRead
[xcb] This is most likely caused by a broken X extension library
[xcb] Aborting, sorry about that.
python: ../../src/xcb_io.c:736: _XRead: Assertion `!xcb_xlib_too_much_data_requested' failed.
Fatal Python error: Aborted

Thread 0x00007f9e4da97700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 359 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 621 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 1080 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9dcbbd5700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 359 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/operators/cacheMemoryManager.py", line 241 in _wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/operators/cacheMemoryManager.py", line 159 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9def7fe700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9deffff700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e10ff9700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e117fa700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e11ffb700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e127fc700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e12ffd700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e137fe700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e13fff700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e28add700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e292de700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e29adf700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e2a2e0700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e2aae1700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e2b322700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Thread 0x00007f9e4e298700 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 340 in wait
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 208 in _get_next_job
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 157 in run
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 810 in __bootstrap_inner
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/lib/python2.7/threading.py", line 783 in __bootstrap

Current thread 0x00007f9e757ab740 (most recent call first):
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/ilastik/ilastik/shell/gui/startShellGui.py", line 69 in startShellGui
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/ilastik/ilastik_main.py", line 107 in main
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/ilastik/ilastik.py", line 104 in main
  File "/home/viola/Desktop/ilastik-1.2.0rc5-Linux/ilastik-meta/ilastik/ilastik.py", line 111 in <module>
./run_ilastik.sh: line 57: 22581 Aborted                 (core dumped) ${PREFIX}/bin/python ${PREFIX}/ilastik-meta/ilastik/ilastik.py "$@"
```

**Comment by imagejan** [comment]
> I user reported on the mailing list ...

Where is this suspicious mailing list?
Can you please subscribe me?
Because this: http://ilastik.org/community.html is _still_ not working for me.

**Comment by stuarteberg** [comment]
>Where is this suspicious mailing list?

This issue was reported to team@ilastik.org

>Can you please subscribe me?
Because this: http://ilastik.org/community.html is still not working for me.

Sorry about that.  The team@ilastik.org mailing list is not publicly subscribable or viewable.  We'll try to get the ilastik-user list back up and running.

**Comment by stuarteberg** [comment]
Issue was related to OpenGL. The user reports that it is fixed as of 1.2.0rc7 (which disables OpenGL again).
